.travis.yml

```diff
@@ -32,13 +32,8 @@ jobs:
     - name: "Quality checks"
       stage: "prebuild"
```

Comment:
I think we can take advantage of this by removing the stages to speed up CI

Reply:
I followed cozy-store, what do you think https://github.com/cozy/cozy-store/blob/master/.travis.yml

Reply:
Nice 👍